components/badges/index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Badges | Horsemaker's UI</title>
+		<link
+			href="https://fonts.googleapis.com/css2?family=Material+Icons"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="../../default.css" />
+		<link rel="stylesheet" href="../../main.css" />
+		<link rel="stylesheet" href="style.css" />
+		<link rel="stylesheet" href="badge.css" />
+	</head>
+	<body>
+		<div class="header">
+			<a class="header-link" href="/index.html"
+				><img
+					class="avatar-sm"
+					src="../../assets/horsemaker.svg"
+					alt="🦄"
+				/>
+				<h2>Horsemaker's UI</h2></a
+			>
+		</div>
+		<div class="main badges">
+			<h1>Badges</h1>
+			<p>
+				Badges are being used to display the status information or the
+				notification count.
+			</p>
+			<!-- Text with a badge -->
+			<p>Text with a badge</p>
+			<div class="playground">
+				<span class="badge">
+					Alerts
+					<span
+						class="badge-content badge-small badge-right badge-orange"
+						>4</span
+					>
+				</span>
+			</div>
+			<!-- Text with large badge -->
+			<p>Text with large badge</p>
+			<div class="playground">
+				<span class="badge">
+					Alerts
+					<span
+						class="badge-content badge-large badge-right badge-purple"
+						>4</span
+					>
+				</span>
```

Comment:
You can write in 
`
<span class="badge">
        Alerts
      <span class="badge-content badge-large badge-right badge-purple">4</span>
</span>

It would be easy to read for the people who reviews the code
`

Reply:
thanks for pointing this out!